server/block/slime.go

```diff
@@ -0,0 +1,21 @@
+package block
+
+// Slime is a translucent block that entities can bounce on.
+type Slime struct {
```

Comment:
This also should implement `block.LightDiffuser` with `LightDiffusionLevel()` returning the correct light diffusion level